primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js

```diff
@@ -1201,7 +1193,7 @@
 
             // try and strip specific language from nl_BE to just nl
             if (!locale) {
-                var localeKey = cfgLocale ? cfgLocale : PrimeFaces.settings.locale;
+                var localeKey = cfgLocale || PrimeFaces.settings.locale;
```

Comment:
this localKey var could be renamed since it's a boolean

Reply:
This actually is the value "en_US" either using the local config or global locale value before it gets split to "en".

Reply:
ohh